libnavigation-core/api/current.txt

```diff
@@ -173,6 +176,117 @@ package com.mapbox.navigation.core.arrival {
+    method public String getAddress();
+    method public com.mapbox.navigation.core.copilot.HistoryPoint? getCoordinates();
+    method public String getId();
+    method public String getName();
+    method public java.util.List<com.mapbox.navigation.core.copilot.HistoryRoutablePoint>? getRoutablePoint();
+    property public final String address;
+    property public final com.mapbox.navigation.core.copilot.HistoryPoint? coordinates;
+    property public final String id;
+    property public final String name;
+    property public final java.util.List<com.mapbox.navigation.core.copilot.HistoryRoutablePoint>? routablePoint;
+  }
+
+  @Keep public final class SearchResultUsed {
+    ctor public SearchResultUsed(String provider, String id, String name, String address, com.mapbox.navigation.core.copilot.HistoryPoint coordinates, java.util.List<com.mapbox.navigation.core.copilot.HistoryRoutablePoint>? routablePoint);
+    method public String component1();
+    method public String component2();
+    method public String component3();
+    method public String component4();
+    method public com.mapbox.navigation.core.copilot.HistoryPoint component5();
+    method public java.util.List<com.mapbox.navigation.core.copilot.HistoryRoutablePoint>? component6();
+    method public com.mapbox.navigation.core.copilot.SearchResultUsed copy(String provider, String id, String name, String address, com.mapbox.navigation.core.copilot.HistoryPoint coordinates, java.util.List<com.mapbox.navigation.core.copilot.HistoryRoutablePoint>? routablePoint);
+    method public String getAddress();
+    method public com.mapbox.navigation.core.copilot.HistoryPoint getCoordinates();
+    method public String getId();
+    method public String getName();
+    method public String getProvider();
+    method public java.util.List<com.mapbox.navigation.core.copilot.HistoryRoutablePoint>? getRoutablePoint();
+    property public final String address;
+    property public final com.mapbox.navigation.core.copilot.HistoryPoint coordinates;
+    property public final String id;
+    property public final String name;
+    property public final String provider;
+    property public final java.util.List<com.mapbox.navigation.core.copilot.HistoryRoutablePoint>? routablePoint;
+  }
+
+  public final class SearchResultUsedEvent extends com.mapbox.navigation.core.copilot.HistoryEvent {
+    ctor public SearchResultUsedEvent(com.mapbox.navigation.core.copilot.SearchResultUsed searchResultUsed);
+    method public com.mapbox.navigation.core.copilot.SearchResultUsed component1();
+    method public com.mapbox.navigation.core.copilot.SearchResultUsedEvent copy(com.mapbox.navigation.core.copilot.SearchResultUsed searchResultUsed);
+    method public com.mapbox.navigation.core.copilot.SearchResultUsed getSearchResultUsed();
+    property public final com.mapbox.navigation.core.copilot.SearchResultUsed searchResultUsed;
+  }
+
+  @Keep public final class SearchResults {
```

Comment:
Do you want to start with @Experimental annotations while it is being integrated further?

Just in case there are some new learnings from wider adoption, you'll at least be able to make minor tweaks

Reply:
Yeah, good point. Was planning to mark any Copilot API as `@Experimental` until we go GA 👍 

Opened this as a `Draft` to get some initial feedback from you but yeah, APIs are going to be `@Experimental` for now.

Reply:
Added `@ExperimentalPreviewMapboxNavigationAPI` annotations to all public entities 👍